local/compose/create.go

```diff
@@ -335,6 +368,15 @@ func getVolumesFrom(project *types.Project, volumesFrom []string) ([]string, []s
 
 }
 
+func getNetworkFrom(networkFrom string) string {
+	baseService := ""
+	if strings.HasPrefix(networkFrom, "service:") {
```

Comment:
`"service:"` should be declared a const in compose-go

Reply:
Depends on merge of https://github.com/compose-spec/compose-go/pull/109